frontend/src/App.test.js

```diff
@@ -1,4 +1,5 @@
 import { render, screen } from '@testing-library/react'
+import { expect, test } from '@jest/globals';
 import App from './App'
 
 test('renders learn react link', () => {
```

Comment:
When you done moving routing to index. then this file can be deleted

Reply:
✔️